install/ubuntu-installation-guide.md

```diff
@@ -0,0 +1,53 @@
+# Installing Kata Containers on Ubuntu
+
+Note:
+Kata Containers is available for Ubuntu\* **16.04** and **17.10**.
+
+This step is only required in case Docker is not installed on the system.
+1. Install the latest version of Docker with the following commands:
+
+```
+$ sudo -E apt-get -y install apt-transport-https ca-certificates wget software-properties-common
+$ wget -qO - https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+$ sudo -E add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
```

Comment:
Hard-coded architecture. It's difficult to know what the best approach is here. I think maybe removing the `[arch=...]` part entirely and adding a warning/assumption at the top of the document stating that we only build for x86_64 currently would work. That way, when we enable other architectures, we can update/remove that introductory text and avoid having to modify the apt config lines like this.

```
sudo -E add-apt-repository "deb [arch=amd64]
```